tests/unit_tests/test_UpdatedSynDataset_target_creator.py

```diff
@@ -0,0 +1,242 @@
+        self.test_image11 = TargetCreator.create_specified_single_target(11, "K", 500, 2, (255, 0, 0, 255), (0, 0, 0, 255))
+        self.test_image12 = TargetCreator.create_specified_single_target(12, "L", 500, 2, (255, 0, 0, 255), (0, 0, 0, 255))
+        self.test_image13 = TargetCreator.create_specified_single_target(13, "M", 600, 2, (0, 255, 0, 255), (255, 0, 0, 255))
+        self.test_image14 = TargetCreator.create_specified_single_target(14, "N", 600, 2, (0, 255, 0, 255), (255, 0, 0, 255))
+        self.test_image15 = TargetCreator.create_specified_single_target(15, "O", 600, 2, (0, 255, 0, 255), (255, 0, 0, 255))
+        self.test_image16 = TargetCreator.create_specified_single_target(16, "P", 600, 2, (0, 255, 0, 255), (255, 0, 0, 255))
+        self.test_image17 = TargetCreator.create_specified_single_target(17, "Q", 700, 3, (0, 0, 255, 255), (0, 255, 0, 255))
+        self.test_image18 = TargetCreator.create_specified_single_target(18, "R", 700, 3, (0, 0, 255, 255), (0, 255, 0, 255))
+        self.test_image19 = TargetCreator.create_specified_single_target(19, "S", 700, 3, (0, 0, 255, 255), (0, 255, 0, 255))
+        self.test_image20 = TargetCreator.create_specified_single_target(20, "T", 700, 3, (0, 0, 255, 255), (0, 255, 0, 255))
+        self.test_image21 = TargetCreator.create_specified_single_target(21, "U", 800, 3, (255, 255, 0, 255), (0, 0, 255, 255))
+        self.test_image22 = TargetCreator.create_specified_single_target(22, "V", 800, 3, (255, 255, 0, 255), (0, 0, 255, 255))
+        self.test_image23 = TargetCreator.create_specified_single_target(23, "W", 800, 3, (255, 255, 0, 255), (0, 0, 255, 255))
+        self.test_image24 = TargetCreator.create_specified_single_target(24, "X", 800, 3, (255, 255, 0, 255), (0, 0, 255, 255))
+        self.test_image25 = TargetCreator.create_specified_single_target(25, "Y", 900, 3, (255, 0, 255, 255), (255, 255, 0, 255))
+        self.test_image26 = TargetCreator.create_specified_single_target(26, "Z", 900, 3, (255, 0, 255, 255), (255, 255, 0, 255))
+        self.test_image27 = TargetCreator.create_specified_single_target(27, "A", 900, 3, (255, 0, 255, 255), (255, 255, 0, 255))
+        self.test_image28 = TargetCreator.create_specified_single_target(28, "B", 900, 3, (255, 0, 255, 255), (255, 255, 0, 255))
+        self.test_image29 = TargetCreator.create_specified_single_target(29, "C", 1000, 3, (0, 255, 255, 255), (255, 0, 255, 255))
+        self.test_image30 = TargetCreator.create_specified_single_target(30, "D", 1000, 3, (0, 255, 255, 255), (255, 0, 255, 255))
+        self.test_image31 = TargetCreator.create_specified_single_target(31, "E", 1000, 3, (0, 255, 255, 255), (255, 0, 255, 255))
+        self.test_image32 = TargetCreator.create_specified_single_target(32, "F", 1000, 3, (0, 255, 255, 255), (255, 0, 255, 255))
+
+    def test_create_specified_single_target(self):
```

Comment:
Should this all be one unit test? Or can we make a new unit test for each shape that you test? I think it makes more sense to split it up.